Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/validation/Validation.java

```diff
@@ -0,0 +1,68 @@
+package pg.autyzm.friendly_plans.validation;
+
+import android.widget.EditText;
+import java.util.regex.Pattern;
+
+/***
+ * Original of validation class is here:
+ * https://tausiq.wordpress.com/2013/01/19/android-input-field-validation/
+ */
+public class Validation {
+
+    // Regular Expression
+    private static final String NAME_REGEX = "^[a-zA-Z0-9_. -]*$";
+    private static final String NUM_ONLY_REGEX = "^[0-9]*$";
+
+    // Error Messages
+    private static final String WRONG_NAME_MSG = "@string/only_letters_msg";
+    private static final String DUPLICATED_NAME_MSG = "@string/name_exist_msg";
+    private static final String ONLY_NUMBERS_MSG = "@string/only_numbers_msg";
+    private static final String NOT_EMPTY_MSG = "@string/not_empty_msg";
```

Comment:
@mikolevy not completed. Your or Gosia's help required. Lets do screen sharing and fix it together.